src/docs/configuration/merging/README.md

```diff
@@ -176,6 +176,18 @@ tasks.named('shadowJar', com.github.jengelman.gradle.plugins.shadow.tasks.Shadow
 }
 ```
 
+## Merging Log4j2 Plugin Cache Files (`Log4j2Plugins.dat`)
+
+`Log4j2PluginsCacheFileTransformer` is a `Transformer` that merges `META-INF/org/apache/logging/log4j/core/config/plugins/Log4j2Plugins.dat` plugin caches from all the jars
```

Comment:
```suggestion
`Log4j2PluginsCacheFileTransformer` is a `Transformer` that merges [`Log4j2Plugins.dat` plugin descriptors](https://logging.apache.org/log4j/2.x/manual/plugins.html#plugin-registry) from all the jars
```

_Nit_: personally I prefer to call the "descriptors" instead of "caches", although "cache" was the original term used.

Reply:
https://logging.apache.org/log4j/2.x/manual/plugins.html#plugin-registry

This link indicates "Registering plugins" instead of "Log4j2Plugins.dat plugin descriptors" self, I'm worried there might be misleading here?

Reply:
You might be right.
Currently we don't have a better reference for that, just the "registering plugins" section and some [answers in the FAQ](https://logging.apache.org/log4j/2.x/faq.html#plugin-descriptors).

Maybe you could point at the [Shading/Shadowing FAQ entry](https://logging.apache.org/log4j/2.x/faq.html#shading), although that sections almost immediately points back at your documentation. :wink:

Reply:
Alright. I'll leave the file path here just for now.